### PR TITLE
Mark french

### PR DESCRIPTION
**Pull request by juarezfrench**
Finished cards.  MVP!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,15 +28,22 @@ Demonstrate your understanding of this week's concepts by answering the followin
...
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager
 
-1. What is the DOM?
+1. What is the DOM? An abstaction that represents the elements that make up the website, including the HTML, styling and javascript. By looking at them in the DOM, we can analyze and manipulate the DOM instead of the individual pieces.
 
 2. What is an event?
 
+An event is a specific action, usually, but not always initiated by the user that will trigger a component or module that performs a specific action or function.  An example of an event would be a click on a button that saves or minimizes the page.
+
 3. What is an event listener?
```
Comment (thereactgirl) [inline README.md:37]: Perfect explanation

File: README.md [hunk 2]
```diff
@@ -28,15 +28,22 @@ Demonstrate your understanding of this week's concepts by answering the followin
...
 
+An event listener is a code module or component that is listening for the event and is executed when it detects the event. 
+
```
Comment (thereactgirl) [inline README.md:40]: Almost but not quite. An event listener is not a component it's actually a function that you can put on an element that takes in an event as a parameter

File: components/Cards/index.js [hunk 3]
```diff
@@ -17,3 +17,64 @@
...
+
+    categories.forEach(cat => {
+      articleData.forEach(article => {
+        if (articleData[0][0] == cat) {
+          makeCards(article);
+        }
+      });
+    });
+  })
+
```
Comment (thereactgirl) [inline components/Cards/index.js:43]: Nice work pulling in data and looping through it to make your articles!

File: components/Cards/index.js [hunk 4]
```diff
@@ -17,3 +17,64 @@
...
+  author.appendChild(imgCont);
+  imgCont.appendChild(authImg);
+  author.appendChild(byLine);
+
+  cards.classList.add("card");
+  headLn.classList.add("headline");
+  author.classList.add("author");
+  imgCont.classList.add("img-container");
+  authImg.classList.add("img")
+
```
Comment (thereactgirl) [inline components/Cards/index.js:68]: Perfect function that creates elements structures them and adds styles!

**Review by thereactgirl — COMMENTED** [review]
Awesome job on this project! You completed MVP in the allotted amount of time. I also found your comments very helpful. Your code is clean and easy to follow. 2 stars :)

**Comment by juarezfrench** [comment]
Thank you very much! Sorry for the late response - I've been having all kinds of computer issues! I really appreciate your insight!

**Pull request merged**